Microsoft.Toolkit.Uwp.UI.Controls/RangeSelector/RangeSelector.cs

```diff
@@ -623,50 +623,18 @@ private static void RangeMaxChangedCallback(DependencyObject d, DependencyProper
 
         private static void UpdateToolTipText(RangeSelector rangeSelector, TextBlock toolTip, double newValue)
         {
-            if (rangeSelector.StepFrequency < 1)
-            {
-                toolTip.Text = string.Format("{0:0.00}", newValue);
-            }
-            else
-            {
-                toolTip.Text = string.Format("{0:0}", newValue);
-            }
-        }
-
-        private static void UpdateMinimumDisplayText(double newValue, RangeSelector rangeSelector)
-        {
-            // Safety check in case the template is missing the _minValueText element
-            if (rangeSelector._minValueText == null)
-            {
-                return;
-            }
-
-            if (rangeSelector.StepFrequency < 1)
-            {
-                rangeSelector._minValueText.Text = string.Format("{0:0.00}", newValue);
-            }
-            else
-            {
-                rangeSelector._minValueText.Text = string.Format("{0:0}", newValue);
-            }
+            toolTip.Text = string.Format("{0:0.##}", newValue);
         }
 
-        private static void UpdateMaximumDisplayText(double newValue, RangeSelector rangeSelector)
+        private static void UpdateDisplayText(double newValue, TextBlock valueDisplayTextblock)
```

Comment:
This causes a weird "issue" where the size of the element changes as the value changes and causes the control to look jittery. I think we need to keep the size somewhat constant and always show the decimal points if the StepFrequency is a fraction.

Reply:
[This](https://github.com/Microsoft/WindowsCommunityToolkit/pull/2154/files#diff-3b81f5ca9b63aec6a53a54cebd02b969R193) tries to mitigate that, but 50.25 is longer than 100. Even 10.5 is longer than 100

Reply:
I would again argue that these should be removed. The Slider control doesn't have this. Leave it to the dev to put values in and ensure the size is what they expect

Reply:
Even showing the decimal places always is going to cause jumping

Reply:
As part of this PR or a different one?

Reply:
This PR is fine